dali/ft/filecopy.cpp

```diff
@@ -2973,7 +2985,10 @@ void FileSprayer::updateTargetProperties()
                      (stricmp(aname,"@eclCRC")==0)||
                      (stricmp(aname,"@formatCrc")==0)||
                      (stricmp(aname,"@owner")==0)||
-                     ((stricmp(aname,FArecordCount)==0)&&!gotrc))
+                     ((stricmp(aname,FArecordCount)==0)&&!gotrc) ||
+                     ((stricmp(aname,"@blockCompressed")==0)&&copyCompressed) ||
```

Comment:
This looks suspiciously like some code that should be in the dafile code - copyAttribute(name, copyCompression); - otherwise when new attributes are added they may not be copied.  The whole copy should probably 

Probably a different pull request though.